apps/emqx_bridge_http/src/emqx_bridge_http_connector.erl

```diff
@@ -519,15 +520,19 @@ trace_rendered_action_template(ActionId, Method, NRequest, Timeout) ->
             emqx_trace:rendered_action_template(
                 ActionId,
                 #{
+                    host => Host,
                     path => Path,
                     method => Method,
                     headers => emqx_utils_redact:redact_headers(Headers),
```

Comment:
this can be made lazy too ?

Reply:
Fixed: [ff09f14](https://github.com/emqx/emqx/pull/12916/commits/ff09f1419142e5dad089bcb8301710be8d836da6)